cypress/e2e/signIn.cy.js

```diff
@@ -1,34 +1,32 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
-
 import SignInPageObject from '../support/pages/signIn.pageObject';
 import HomePageObject from '../support/pages/home.pageObject';
 
-const signInPage = new SignInPageObject();
-const homePage = new HomePageObject();
+const SignInPage = new SignInPageObject();
+const HomePage = new HomePageObject();
 
 describe('Sign In page', () => {
   let user;
 
   before(() => {
     cy.task('db:clear');
+  });
+
+  beforeEach(() => {
     cy.task('generateUser').then((generateUser) => {
       user = generateUser;
     });
+    SignInPage.visit();
   });
```

Comment:
The 'beforeEach' hook generates a user but does not register it. Ensure that the user is registered before running tests that require the user to exist in the system.